migrations/main.go

```diff
@@ -22,33 +24,18 @@ Usage:
 
 func main() {
 	gotenv.Load()
-	var env string
-	var db *pg.DB
-
-	flag.StringVar(
-		&env,
-		"env",
-		"dev",
-		"Run migration for specified environment")
 
 	flag.Usage = usage
 	flag.Parse()
 
-	if env == "test" {
-		db = pg.Connect(&pg.Options{
-			User:     os.Getenv("TEST_DB_USERNAME"),
-			Password: os.Getenv("TEST_DB_PASSWORD"),
-			Database: os.Getenv("TEST_DATABASE"),
-		})
-	} else {
-		db = pg.Connect(&pg.Options{
-			User:     os.Getenv("DB_USERNAME"),
-			Password: os.Getenv("DB_PASSWORD"),
-			Database: os.Getenv("DATABASE"),
-		})
-	}
+	config := utilities.GetDatabaseCredentials()
+	db := utilities.Connect(config)
```

Comment:
@tomipaul If these two functions are in the same file, why not create a function that uses the two functions in the same file, so you can just call one function that does both things

Reply:
@mikey2020 that's because I don't want a tight coupling. I envision we might need to update /change the implementation of these functions in the not too distant future. I want to be able to treat them like two separate functionalities in the caller so that I can do one without the other e.g get credentials from a different source and use the connect utility to connect to a db.